### ISSUE TITLE
Avoid relying on 'minMaxPos'

### ISSUE THREAD

**Issue by pneuvial** [pneuvial/c3co#17]
Currently 'createZdf' requires an argument 'minMaxPos', a matrix containing min and max position for each chromosome. This should really be taken care of by the package, not by the user. 

If we add to the slot 'bkp' of a fitC3co object a "first breakpoint" at the first position and a "last breakpoint" at the last position, then the object will be selfcontained.

**Comment by pneuvial** [comment]
Fixed [develop 9caaf42]

**Comment by pneuvial** [comment]
Side effect: we're now plotting the horizontal lines twice in Zplot, cf

```
    gg <- gg + geom_step(direction="hv", lwd=1)
    gg <- gg + geom_segment(aes_(xend=~end, yend=~CopyNumber), lwd=1) 
```
